store/tikv/client.go

```diff
@@ -254,7 +256,10 @@ func (c *rpcClient) SendRequest(ctx context.Context, addr string, req *tikvrpc.R
 	var first *coprocessor.Response
 	first, err = copStream.Recv()
 	if err != nil {
-		return nil, errors.Trace(err)
+		if errors.Cause(err) != io.EOF {
+			return nil, errors.Trace(err)
+		}
+		log.Info("copstream returns nothing for the request.")
```

Comment:
Is it expected for tikv to close the stream directly?
If so, I think `Debug` is better, otherwise I think `Warn` is better.

Reply:
I'll change it to debug level